### PR TITLE
cpu, sam3: fix periph/dac poweron

### PR DESCRIPTION
**Pull request by smlng**
factored out from #7995, as requested.

fixes poweron function for DAC of SAM3 cpu.

### CODE REVIEW

File: cpu/sam3/periph/dac.c [hunk 1]
```diff
@@ -64,8 +64,8 @@ void dac_poweron(dac_t line)
 {
     assert(line < DAC_NUMOF);
 
-    PMC->PMC_PCER1 = (1 << (ID_DACC - 32));
-    DACC->DACC_CHER = PMC_BIT;
+    DACC->DACC_CHER = (1 << line);
+    PMC->PMC_PCER1 = PMC_BIT;
```
Comment (haukepetersen) [inline cpu/sam3/periph/dac.c]: This seems incorrect to me: at least the order is broken now -> the module clock gate bit needs to be enabled before the actual module enable bit...

Comment (smlng) [reply cpu/sam3/periph/dac.c]: right, will fix

**Comment by smlng** [comment]
@haukepetersen fixed order

**Comment by haukepetersen** [comment]
all green -> go

**Pull request merged**